crates/optimism/payload/src/builder.rs

```diff
@@ -486,6 +485,19 @@ where
         blob_gas_used = Some(0);
     }
 
+    let is_holocene = chain_spec.is_fork_active_at_timestamp(
+        OptimismHardfork::Holocene,
+        attributes.payload_attributes.timestamp,
+    );
+
+    if is_holocene {
+        extra_data = get_holocene_extra_data(
+            &attributes,
+            chain_spec.base_fee_params_at_timestamp(attributes.payload_attributes.timestamp),
+        )
+        .map_err(|err| PayloadBuilderError::other(err))?;
```

Comment:
I think this should work?
```suggestion
        .map_err(PayloadBuilderError::other)?;
```